src/server/surviveler/assets_load.go

```diff
@@ -7,14 +7,12 @@ package surviveler
 import (
 	"errors"
 	"fmt"
-	"image"
-	"io"
 	"path"
 	"server/resource"
 
-	"golang.org/x/image/bmp"
-
 	log "github.com/Sirupsen/logrus"
+	"github.com/aurelien-rainone/go-detour/detour"
+	"github.com/aurelien-rainone/gogeo/f32/d3"
```

Comment:
I would highly recommend to have those projects in Rookie GameDevs organization. C'mon dude, let's share the glory!